src/dom/position.js

```diff
@@ -99,14 +99,31 @@ export function getOptimalPosition( { element, target, positions, limiter, fitIn
 
 	let { left, top } = getAbsoluteRectCoordinates( bestPosition );
 
-	// (#126) If there's some positioned ancestor of the panel, then its rect must be taken into
-	// consideration. `Rect` is always relative to the viewport while `position: absolute` works
-	// with respect to that positioned ancestor.
 	if ( positionedElementAncestor ) {
 		const ancestorPosition = getAbsoluteRectCoordinates( new Rect( positionedElementAncestor ) );
+		const ancestorComputedStyles = global.window.getComputedStyle( positionedElementAncestor );
 
+		// (#126) If there's some positioned ancestor of the panel, then its `Rect` must be taken into
```

Comment:
This should be full link to the ticket because this is a ticket from different repo https://github.com/ckeditor/ckeditor5-ui-default/issues/126